ctrd/container.go

```diff
@@ -500,3 +500,27 @@ func (c *Client) ResizeContainer(ctx context.Context, id string, opts types.Resi
 
 	return pack.task.Resize(ctx, uint32(opts.Height), uint32(opts.Width))
 }
+
+// WaitContainer waits until container's status is stopped.
+func (c *Client) WaitContainer(ctx context.Context, id string, timeout time.Duration) (int64, error) {
```

Comment:
The `waitContainer` API doesn't require the `timeout` param as you mentioned in the swagger.yml. 

I think we can remove the `timeout` here so that it can make `wait` word clear. WDYT?